tests/aggregation/tests.py

```diff
@@ -441,6 +441,29 @@ def test_aggregate_annotation(self):
         vals = Book.objects.annotate(num_authors=Count("authors__id")).aggregate(Avg("num_authors"))
         self.assertEqual(vals, {"num_authors__avg": Approximate(1.66, places=1)})
 
+    def test_sum_distinct_aggregate(self):
+        """
+        Filters Authors by books, where we have multiple authors for a book.
+        Apply distinct and then the sum aggregation results in an incorrect
+        sum on 1.7.x
+
+        If we distinct the authors, should only have 2. If we distinct and sum,
+        should be 57+46, but on 1.7 it sums author 8 2x, so 57+57+46. The sum
```

Comment:
the comment about how the expected value is reached could go down near the assertion